manifests/service.pp

```diff
@@ -23,8 +23,5 @@
       name       => $service_name,
     }
 
```

Comment:
CI will also want this line removed

Reply:
Thanks for the pointers. Changed it accordingly.